roles/lib_openshift/src/lib/base.py

```diff
@@ -243,7 +266,10 @@ def openshift_cmd(self, cmd, oadm=False, output=False, output_type='json', input
         if self.verbose:
             print(' '.join(cmds))
 
-        returncode, stdout, stderr = self._run(cmds, input_data)
+        try:
+            returncode, stdout, stderr = self._run(cmds, input_data)
```

Comment:
@detiber the discussion you linked to seems to be related to changing the env of the current process. Here the case is different, we are setting up the env for a process we are about to spawn.